src/simsopt/geo/surfaceobjectives.py

```diff
@@ -62,6 +65,9 @@ def d2J_by_dsurfacecoefficientsdsurfacecoefficients(self):
         Calculate the second derivatives with respect to the surface coefficients.
         """
         return self.surface.d2volume_by_dcoeffdcoeff()
+    
+    def dJ_by_dcoils(self):
+        return Derivative()
```

Comment:
This is a bit odd. The Area does not have coils as a dependency, why do we have a `dJ_by_dcoils`?